service/server.py

```diff
@@ -276,6 +300,59 @@ def get_property_address_index_polygon(geometry_data):
     return index_polygon
 
 
+def _get_address_search_response(search_term, page_number):
+    search_term = search_term.upper()
+    if _is_title_number(search_term):
+        return _get_search_by_title_number_response(search_term, page_number)
+    elif _is_postcode(search_term):
+        return _get_search_by_postcode_response(search_term, page_number)
+    else:
+        return _get_search_by_address_response(search_term, page_number)
+
+
+def _render_initial_search_page():
+    return render_template(
+        'search.html',
+        asset_path='/static/',
+        google_api_key=GOOGLE_ANALYTICS_API_KEY,
+        form=TitleSearchForm(),
+        username=current_user.get_id(),
+        )
+
+
+def _get_search_by_title_number_response(search_term, page_number):
+    title_ref = search_term
+    title = get_register_title(title_ref)
+    if title:
+        # If the title exists store it in the session
+        # TODO: stop storing title in session
```

Comment:
It is used - the title details page assumes it's there and returns 404 if it's not. I created a story for this.